tests/model_registry/test_rest_api.py

```diff
@@ -4,13 +4,11 @@
 
 LOGGER = get_logger(name=__name__)
 
-schema = schemathesis.from_pytest_fixture("generated_schema")
+schema = schemathesis.pytest.from_fixture("generated_schema")
 
 
-# TODO: This is a Stateless test due to how the openAPI spec is currently defined in upstream.
-# Once it is updated to support Stateful testing of the API we can enable this to run every time,
-# but for now having it run manually to check the existing failures is more than enough.
-@pytest.mark.skip(reason="Only run manually for now")
+@pytest.mark.fuzzer
```

Comment:
i do not think that sanity tests should be marked as `fuzzer`; sanity tests are a set of tests to check basic functionality.  fuzzing (or in this case tools which are based on it) is a different level of testing

Reply:
if this is part of sanity, then this test must be stable and should not be mark as `fuzzer`
```
Mark tests that use fuzzing and are probably going to generate unanticipated failures.
```

Reply:
Removed the Sanity mark